internal/view/assets/js/component/bookmark.js

```diff
@@ -94,8 +94,9 @@ export default {
 			return this.excerpt !== "" && !this.thumbnailVisible && !this.HideExcerpt;
 		},
 		thumbnailStyleURL() {
+			var cacheBuster = new Date().getTime();
 			return {
-				backgroundImage: `url("${this.imageURL}")`,
```

Comment:
Can we use some properties from the bookmark instead of a random string? This could break the cache unintentially since each time the query string is going to be different. Can we use `?mtime=<bookmark modified time>` ?

Reply:
I always assumed `Bookmark.ModifiedTime` was the datetime when the bookmark was last modified (attributes/archive/readable/ebook) modified, it seems that is not the case. Let me think and get back to you.

Reply:
I ended up not updating you here.

My idea is:
- Refactor the migration system (#876)
- Add a `created` column to the bookmarks with the time the bookmark was first created and prefill it with the current `modified` for existing bookmarks.
- Update `modified` every time the bookmark is updated (either the data, content, archive, etc).

Then, we can use the `modified` as query parameter for the thumbnail.